packages/node/tests/relay/Call.test.ts

```diff
@@ -238,9 +239,10 @@ describe('Call', () => {
     })
 
     describe('connect methods', () => {
+      const PHONE = 'phone' as const
       const _tmpDevices = [
-        { type: 'phone', to: '999', from: '231', timeout: 10 },
-        { type: 'phone', to: '888', from: '234', timeout: 20 }
+        { type: PHONE, to: '999', from: '231', timeout: 10 },
+        { type: PHONE, to: '888', from: '234', timeout: 20 }
```

Comment:
Can you add a test cases for connect a `sip` device please? And the same for `/relay/calling.test.ts` below if you can. Thanks!